src/Console/GeneratePaletteCommand.php

```
@@ -22,115 +21,121 @@ class GeneratePaletteCommand extends Command
-
-	protected function swatchList( array $scale ): string
-	{
-		$parts = array_map( fn( $hex ) => sprintf( '`%s`', $hex ), $scale );
-		return implode( ' ', $parts );
-	}
+    protected $description = 'Generate an accessible color palette from seed colors.';
+
+    public function handle(): int
+    {
+        $primary = $this->option('primary');
+        $secondary = $this->option('secondary');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$secondary'. (undefined)

(UnusedLocalVariable)